File: Geeks For Geeks Solutions/Move all zeroes to end of array/Move-all-zeroes-to-end-of-array.py
class Solution:
    def pushZerosToEnd(self,arr, n):
        last_pos_idx = -1
        for i in range(n):
            if arr[i] > 0:
                arr[last_pos_idx + 1] = arr[i]
                if last_pos_idx + 1 < i: arr[i] = 0
                last_pos_idx += 1
        return arr

File: Geeks For Geeks Solutions/Move all zeroes to end of array/Move-all-zeroes-to-end-of-array.cpp
class Solution{
public:
	void pushZerosToEnd(int arr[], int n) {
	    // code here
	      int count = 0;

   for(int i =0;i<n;i++){

       if(arr[i]!=0){

           arr[count] = arr[i];

           count++;

       }

   }

   for(int i =count;i<n;i++){

       arr[i] = 0;

   }
	}
};
